### PR TITLE
[TradingView] add _get_feed_consumers

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot-Trading/pull/1023
requires https://github.com/Drakkar-Software/OctoBot/pull/2524

### CODE REVIEW

**Comment by GuillaumeDSM** [comment]
ready

File: Trading/Mode/trading_view_signals_trading_mode/resources/TradingViewSignalsTradingMode.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ orders amount syntax</a>.
 - `STOP_PRICE` is the price of the stop order to create. When increasing the position or buying in spot trading, the stop loss will automatically be created once the initial order is filled. When decreasing the position (or selling in spot) using a LIMIT `ORDER_TYPE`, the stop loss will be created instantly. *Orders crated this way are compatible with PNL history.*
 - `TAKE_PROFIT_PRICE` is the price of the take profit order to create. When increasing the position or buying in spot trading, the take profit will automatically be created once the initial order is filled. When decreasing the position (or selling in spot) using a LIMIT `ORDER_TYPE`, the take profit will be created instantly. *Orders crated this way are compatible with PNL history.*
 - `REDUCE_ONLY` when true, only reduce the current position (avoid accidental short position opening when reducing a long position). **Only used in futures trading**. Default is false
+- `TAG` is an identifier to give to the orders to create.
```
Comment (Herklos) [inline Trading/Mode/trading_view_signals_trading_mode/resources/TradingViewSignalsTradingMode.md:39]: :+1:

File: Trading/Mode/daily_trading_mode/daily_trading.py [hunk 2]
```diff
@@ -713,19 +728,24 @@ async def create_new_orders(self, symbol, final_note, state, **kwargs):
...
+                            created_orders.append(updated_limit)
                         # ensure limit order was not instantly filled
-                        if use_stop_orders and updated_limit.is_open():
-                            oco_group = self.exchange_manager.exchange_personal_data.orders_manager \
-                                .create_group(trading_personal_data.OneCancelsTheOtherOrderGroup)
-                            updated_limit.add_to_order_group(oco_group)
+                        if create_stop_only or (use_stop_orders and updated_limit and updated_limit.is_open()):
+                            oco_group = None
+                            if updated_limit:
+                                oco_group = self.exchange_manager.exchange_personal_data.orders_manager \
```
Comment (Herklos) [inline Trading/Mode/daily_trading_mode/daily_trading.py:746]: :+1:

**Review by Herklos — APPROVED** [review]
nice feature

**Comment by GuillaumeDSM** [comment]
merged by https://github.com/Drakkar-Software/OctoBot-Tentacles/pull/1174

**Pull request closed without merging**